### PR TITLE
Google+ counter sometimes has a dot, delimiting thousands.

### PR DESCRIPTION
**Pull request by avlukanin**
This is not a float number. Sorry, I don't know how to use Grunt to minify the script.

### CODE REVIEW

File: src/social-likes.js [hunk 1]
```diff
@@ -498,6 +498,9 @@
 		},
 
 		updateCounter: function(number) {
+			if (typeof number == 'string') {
```
Comment (sapegin) [inline src/social-likes.js]: ===

File: src/social-likes.js [hunk 2]
```diff
@@ -498,6 +498,9 @@
...
+				number.replace('.', '');
```
Comment (sapegin) [inline src/social-likes.js]: `'.'` → `/\D/g`

**Comment by sapegin** [comment]
Thanks!

“Sorry, I don't know how to use Grunt to minify the script” — please, never add any non-source code to pull requests.

**Comment by avlukanin** [comment]
OK. This sounds great!

**Comment by sapegin** [comment]
Fixed in adce2e7.

**Pull request closed without merging**